indico/modules/events/surveys/models/surveys.py

```diff
@@ -197,3 +232,23 @@ def close(self):
         if self.state not in (SurveyState.active_and_clean, SurveyState.active_and_answered):
             raise IndicoError("Survey can't be closed")
         self.end_dt = now_utc()
+
+    def send_start_notification(self):
+        if not self.notifications_enabled or self.start_notification_sent:
+            return
+        addresses = self.start_notification_emails
+        if self.notify_participants:
+            addresses += self.event.getRegistrantsList() + self.event.getParticipation().getParticipantsList()
+
+        template_module = get_template_module('events/surveys/emails/start_notification_email.txt', survey=self)
+        send_email(make_email(bcc_list=set(addresses), template=template_module))
+        logger.info('Notifications sent to {}'.format(addresses))
+        self.event.log(EventLogRealm.event, EventLogKind.positive, u'Surveys', u'Survey start notifications has been sent')
```

Comment:
the file uses unicode literals. no need for the `u` prefix. also, "have" for plural.